packages/react-search-ui/src/containers/Results.js

```diff
@@ -4,7 +4,7 @@ import { Result, Results } from "@elastic/react-search-ui-views";
 
 import { withSearch } from "..";
 import { Result as ResultContainer } from ".";
-import { Result as ResultType } from "../types";
+import { ResultWithRequiredId as ResultType } from "../types";
```

Comment:
OK, so just so I'm understanding the shape change:

`FieldValueWrapper` type:
```
{
  raw: 'foo',
  snippet: 'bar'
}
```

the new WithRequiredId type:
```
{
  id: {
    raw: 'foo',
    snippet: 'optional',
  }
}
```

is that right?